views/domain.html.tmpl

```diff
@@ -0,0 +1,19 @@
+<html>
+  <body>
+    {{ if or (eq .Response.State "unknown") (eq .Response.State "") }}
+      Something went wrong - your domain was not submitted to the STARTTLS policy list.
+    {{ end }}
+    {{ if eq .Response.State "unvalidated" }}
+      Thank you for submitting your domain. Please check postmaster@{{ .Response.Name }} to validate that you control the domain.
+    {{ end }}
+    {{ if eq .Response.State "queued" }}
+      Thank you for submitting your domain. Your domain is queued to be reviewed and added to the STARTTLS policy list.
+    {{ end }}
+    {{ if eq .Response.State "failed" }}
+      Your domain does not qualify for addition to the STARTTLS policy list. Please visit {{ .BaseURL }} to review your domain's status and correct any issues.
+    {{ end }}
+    {{ if eq .Response.State "added" }}
+      Your domain is already on the STARTTLS policy list. Thanks for helping to make e-mail more secure.
+    {{ end }}
```

Comment:
I think a lot of these would never actually show up-- we have some short-circuits in `api.Queue` that don't actually return the result, like [here](https://github.com/EFForg/starttls-backend/pull/171/files#diff-651a84f8ad6a38c1f7ccdf63f2692410L257) and [here](https://github.com/EFForg/starttls-backend/pull/171/files#diff-651a84f8ad6a38c1f7ccdf63f2692410L246)